resources/templates/md/pages/about.md

```diff
@@ -3,13 +3,20 @@
  :page-index 1
  :navbar? true}
 
-The Bangalore Clojure Group hosts one of the most active (since 2013) meetups for Clojure in India, each of our meetups are conducted in different locations with different set of speakers every month.Our meetups are scheduled conveniently on every first Saturday of the month.
+The Bangalore Clojure Group hosts one of the most active (since 2010) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
 
-Our meetups usually have talks by clojurists, but we also do organise hackathons on a regular basis. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)
+Our meetups usually have talks by Clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists and functional freaks without any bias (We also welcome Windows users 🙂)
```

Comment:
Consider s/freaks/enthusiasts and put a full stop after "Windows users".

Reply:
Fixed. Removed "Windows users" thing. I wasn't sure if this comment will triiger a different emotion in someone's mind.